test/http.js

```diff
@@ -10,7 +10,7 @@ describe('HTTP APIs', function() {
 
   var emptyResponse = function(res) {
       if (res.text.length !== 0) {
-          error("Not empty response");
+          console.log("Not empty response");
```

Comment:
I have a feeling this should be `throw new Error`, I will need to look into this